tests/packagedcode/test_debian_copyright.py

```diff
@@ -26,7 +26,7 @@
 def check_expected_parse_copyright_file(
     test_loc,
     expected_loc,
-    regen=False,
+    regen=True,
```

Comment:
You need to revert this to "regen=False", the regen True was only for local regeneration ... otherwise the tests are testing nothing. And you would need to commit the updated expected JSON test results too